README.md

```diff
@@ -22,6 +22,15 @@ The genesis config should define the block heights for `berlinBlock`, `londonBlo
     }
 ```
 
+To test EIP-2930, the following memory address and contents are set in the `alloc` section of the genesis config to simplify the test.
+```
+      "0x000000000000000000000000000000000000aaaa" : {
```

Comment:
Please add it to the genesis file also

Reply:
Thanks to point it out. This is incorrect actually, the address here should be `0x7B31188CA9C1374AC9174C3D1F23F98180CBB67C` which is added to the genesis. I will fix it.